src/components/utils/ErrorBoundary.js

```diff
@@ -13,7 +13,7 @@ class ErrorBoundary extends Component {
   }
   render() {
     if (this.state.hasError) {
-      return <h1>Whoops! Our bad, it's probaby nothing. Try again!</h1>
+      return "<h1>Whoops! Our bad, it's probaby nothing. Try again!</h1>"
```

Comment:
This will actually render the `<h1>`, so we want to keep it HTML rather than a string